general_tools/file_utils.py

```diff
@@ -99,8 +99,8 @@ def load_json_object(file_name, default=None):
 
 
 def read_file(file_name):
-    contents = codecs.open(file_name, 'r', encoding='utf-8').read()
-    return contents
+    with codecs.open(file_name, 'r', encoding='utf-8') as f:
```

Comment:
Just a suggestions: I prefer to use `encoding = 'utf-8-sig'` because it properly handles files with a byte order mark.